src/components/Dashboard/AccountBanner.js

```diff
@@ -186,14 +184,15 @@ const BannerWithProbability = ({ activeBalance }) => {
       `}
     >
       <span css="margin-right: 8px">
+        You’ll be drafted, on average
         <span
           css={`
             color: ${theme.accent};
           `}
         >
-          {probabilityText} probability{' '}
+          {` 1 on every ${chances} times `}
         </span>
-        to be drafted
+        during this term
```

Comment:
I wonder if "during this term" is really needed; it makes it sound like there are actually going to be X disputes created this term.